src/scripts/main.js

```diff
@@ -0,0 +1 @@
+// This puts a <style> tag in <head> of page with transpiled css.
```

Comment:
I have a sneeky suspicion you might need to `require('../stylesheets/main.scss')`

Like I did in [bootstrap webpack](https://github.com/richardgill/webpack-bootstrap-seed/blob/master/src/scripts/main.js)